authenticode/checksum.go

```diff
@@ -29,10 +29,10 @@ type PECOFFBinary struct {
 	// DataDirectory for the Certificate table
 	Datadir pe.DataDirectory
 	// Reader with the hashable bytes
-	HashContent  *bytes.Buffer
+	HashContent  SizeReaderAt
```

Comment:
I think we can just change this to be private. I don't think people need access to this?

Reply:
👍